### PR TITLE
fix: use MP money fields for My Bids floor selling price and online asking price (PURCHASE-2171)

### PR DESCRIPTION
**Pull request by starsirius**
The type of this PR is: **Bugfix**

This PR resolves [PURCHASE-2171] and depends on https://github.com/artsy/metaphysics/pull/2805.

### Description

Following https://github.com/artsy/metaphysics/pull/2805, this migrates `floorSellingPrice` and `onlineAskingPrice` to Metaphysics `Money` type (instead of the deprecated Causality money type).

Migration steps:

- [x] (artsy/causality#848) Add the new raw cents fields (e.g. `floorSellingPriceCents`) and deprecate the Causality `Money` type fields (e.g. `floorSellingPrice`).
- [x] (https://github.com/artsy/ohm/pull/707) Migrate Ohm to use the new cents fields.
- [x] (https://github.com/artsy/causality/pull/852) Remove the Causality `Money` type fields (e.g. `floorSellingPrice`).
- [x] (https://github.com/artsy/metaphysics/pull/2805) Update MP to stitch in MP `Money` type fields.
- [x] (This PR) Update Eigen to use the new fields.

Screen Shot 2020-11-29 at 11 46 50 PM

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[PURCHASE-2171]: https://artsyproduct.atlassian.net/browse/PURCHASE-2171

### CODE REVIEW

File: data/schema.graphql [hunk 1]
```diff
@@ -1811,13 +1814,14 @@ type AuctionsLotStandingEdge {
   node: AuctionsLotStanding!
 }
 
-# A lot in a sale
+# The state of a lot
 type AuctionsLotState {
   # total number of bids placed on the lot
   bidCount: Int!
+  floorSellingPrice: Money
```
Comment (starsirius) [inline data/schema.graphql:1821]: I'm manually updating the schema here as I don't want to wait for the auto-generated PR after the [Metaphysics PR](https://github.com/artsy/metaphysics/pull/2805) is merged. I'll keep an eye on it if that causes issues.

File: src/lib/Scenes/MyBids/Components/ActiveLot.tsx [hunk 2]
```diff
@@ -49,10 +49,10 @@ export const ActiveLotFragmentContainer = createFragmentContainer(ActiveLot, {
         reserveStatus
         soldStatus
         askingPrice: onlineAskingPrice {
```
Comment (ashleyjelks) [inline src/lib/Scenes/MyBids/Components/ActiveLot.tsx:51]: I don't think 'askingPrice' is being used anywhere.

**Review by ashleyjelks — APPROVED** [review]

**Comment by ashleyjelks** [comment]
Looks good!

**Comment by starsirius** [comment]
Merging and giving it a try. Thanks for the review!

File: src/lib/Scenes/MyBids/Components/ActiveLot.tsx (hunk 2 above)
Comment (starsirius) [reply src/lib/Scenes/MyBids/Components/ActiveLot.tsx:51]: Good catch! @erikdstock can you confirm if online asking price is needed for active lot and/or closed lot?

**Pull request merged**